src/Symfony/Contracts/HttpClient/HttpClientInterface.php

```diff
@@ -35,9 +35,9 @@ interface HttpClientInterface
         'body' => '',           // array|string|resource|\Traversable|\Closure - the callback SHOULD yield a string
                                 //   smaller than the amount requested as argument; the empty string signals EOF; when
                                 //   an array is passed, it is meant as a form payload of field names and values
-        'json' => null,         // array|\JsonSerializable - when set, implementations MUST set the "body" option to
-                                //   the JSON-encoded value and set the "content-type" headers to a JSON-compatible
-                                //   value if they are not defined - typically "application/json"
+        'json' => null,         // mixed - when set, implementations MUST set the "body" option to the JSON-encoded
```

Comment:
when defined

Reply:
Changed it to "if set". We either have to remove the default option completely to support null, or null is inconsequential, making "set" the correct term.